ch_tools/chadmin/cli/table_group.py

```diff
@@ -242,20 +243,41 @@ def columns_command(ctx, database_name, table_name):
     default=False,
     help="Enable dry run mode and do not perform any modifying actions.",
 )
+@option(
+    "--detached",
+    is_flag=True,
+    help="Delete detached tables (with nonreplicated engine).",
+)
 @pass_context
 def delete_command(
     ctx,
     _all,
     on_cluster,
     sync_mode,
     dry_run,
+    detached,
     **kwargs,
 ):
     """
     Delete one or several tables.
     """
     cluster = get_cluster_name(ctx) if on_cluster else None
 
+    if detached:
+        logging.info("delete detached table")
+
+        database_name = kwargs.get("database_pattern")
```

Comment:
Suggest to add additional arguments for `database_name` and `table_name` to avoid using patterns allowing multi-selection as exact names. 

```
@argument("database_name", metavar="DATABASE", required=False)
@argument("table_name", metavar="TABLE", required=False)
```

Then deleting detached table will looks like 
```
table delete --detached test_db test_table
```

Also it would be good to forbid using inapplicable options when deleting detached tables. It can be done something like this: 
```
@constaint(If('detached', then=accept_none), ['database_pattern', 'exclude_database_pattern', ...
```
https://cloup.readthedocs.io/en/stable/pages/constraints.html

Reply:
Thank you, good things.